integrations/gcp/gcp_core/search/resource_searches.py

```diff
@@ -26,6 +32,17 @@
     parse_latest_resource_from_asset,
 )
 
+_RETRIABLE_ERROR_TYPES = (TooManyRequests, ServiceUnavailable)
+
+
+def _is_retryable(exc: Exception) -> bool:
+    return isinstance(exc, _RETRIABLE_ERROR_TYPES)
+
+
+retry_policy = AsyncRetry(
+    predicate=_is_retryable, initial=1.0, maximum=60.0, multiplier=2.0, deadline=300.0
```

Comment:
Extract all of these as constants with meaningful names

Reply:
sure